tldr/cfg_extractor.py

```
@@ -831,6 +958,129 @@ def _visit_ruby_call(self, node):
                 if child.is_named:
                     self._visit_node(child)
 
+    def _visit_r_call(self, node):
```

Ruff (0.14.13)

Missing return type annotation for private function `_visit_r_call`

Add return type annotation: `None`

(ANN202)